templates/post_note.tpl.html

```diff
@@ -64,12 +64,21 @@
     $().ready(function() {
         $('#clear_note_cc').click(clearExtraRecipients);
         $('#add_email_signature').click(setSignature);
+
+        $('#post_note_form').garlic({
+            getPath: function ( $elem ) {
+              var form = $elem.parents('form');
+              return form.find('input[name=issue_id]').val() + '_' +
+                      form.find('input[name=parent_id]').val() +
+                      form.data('sup-id') + $elem.attr( 'name' );
+            }
```

Comment:
That was my plan, but I missed it. using join is much cleaner looking as
well. Done

On Thu, Feb 18, 2016 at 5:18 AM, Elan Ruusamäe <notifications@github.com>
wrote:

> In templates/post_note.tpl.html
> <https://github.com/eventum/eventum/pull/145#discussion_r53301177>:
>
> > @@ -64,12 +64,21 @@
> >      $().ready(function() {
> >          $('#clear_note_cc').click(clearExtraRecipients);
> >          $('#add_email_signature').click(setSignature);
> > +
> > +        $('#post_note_form').garlic({
> > +            getPath: function ( $elem ) {
> > +              var form = $elem.parents('form');
> > +              return form.find('input[name=issue_id]').val() + '_' +
> > +                      form.find('input[name=parent_id]').val() +
> > +                      form.data('sup-id') + $elem.attr( 'name' );
> > +            }
>
> i think should always specify the separator between elements to avoid same
> id for:
>
>    - parent_id 1 + sup_id 11
>    - parent_id 11 + sup_id 1
>
> [ item1, item2 ].join('_') ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/eventum/eventum/pull/145/files#r53301177>.
>



-- 
Bryan Alsdorf
MariaDB Corporation | http://mariadb.com